src/front/app/components/challenges/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ChallengeList from './ChallengeList';
+import Menu from './Menu';
+import Navigation from './../navigation';
+
+export default () => (
+    <div>
+        < Menu />
```

Comment:
Not sure why eslint has no trailing space after opening tag brace check
As for me we should write ```<Menu />``` instead of ```<     Menu />```, and same for next tags